code/addons/links/package.json

```diff
@@ -82,7 +82,7 @@
   "dependencies": {
     "@storybook/client-logger": "7.0.0-beta.43",
     "@storybook/core-events": "7.0.0-beta.43",
-    "@storybook/csf": "next",
+    "@storybook/csf": "0.0.2--canary.62.5777911.0",
```

Comment:
Was this change for testing purposes or is that change correct?

Reply:
It is.. but this makes it so it doesn't depend on lodash at all

Reply:
Needs a new release of the package, this is effectively a PR-release of `@storybook/csf`